src/utilities/git.jl

```diff
@@ -90,11 +90,12 @@ function git_get_master_branch(master_branch::DefaultBranch)
 end
 git_get_master_branch(master_branch::AbstractString) = master_branch
 
-function git_remote_add_and_seturl(remote_name::AbstractString, url::AbstractString)
-    try
-        run(`git remote add $remote_name $url`)
-    catch
+function git_remote_add_or_seturl(remote_name::AbstractString, url::AbstractString)
+    remotes = split(strip(read(`git remote`, String)), '\n')
```

Comment:
To make testing a little easier, can you factor this out into a separate function named `remote_exists(remote_name::AbstractString)` that returns a Bool?